src/main/webapp/eventScript.js

```diff
@@ -278,8 +278,39 @@ function createEventNoResponse(event) {
   eventDetails.className = "details-display";
   eventDetails.innerText = event.eventDetails;
 
+  const topOfEvent = document.createElement('div');
+  topOfEvent.id = "top-event";
+
+  const creatorName = document.createElement('div');
+  creatorName.id = "event-creator";
+  fetch("/user")
+    .then(response => response.json())
+    .then(users => {
+      for (let i = 0; i < users.length; i++) {
+        if (users[i].id === event.creator) {
+          displayProfilePicture(users[i], document.getElementById("event-creator"), 'profile-pic-small');
```

Comment:
You should be able to replace this call to `document.getElementById` by simply passing the `creatorName` element you instantiated above.